src/main/ui/nametag-label.gd

```diff
@@ -0,0 +1,12 @@
+class_name NametagLabel
+extends Label
+"""
+A label used to display a 'nametag', a box at the top of a chat window with the speaker's name
+"""
+
+"""
+Sets the label text, returning 'false' if it causes the label to exceed max_lines_visible
+"""
+func cram_text(_text: String) -> bool:
+	text = _text
+	return get_line_count() <= max_lines_visible
```

Comment:
_(refactor)_ This class is a little anemic; maybe we can push some logic from ChatUI into it.

Reply:
Deleted class. Its code has been absorbed into `NametagManager`.